server/controller/answers.js

```diff
@@ -0,0 +1,22 @@
+const Question = require('../models/question.js');
+const Answer = require('../models/answer.js');
+
+exports.addAnswer = async (ctx) => {
+  await Question.findById(ctx.request.body.question_id)
+    .then((res) => {
+      if (res) {
+        const answer = new Answer(ctx.request.body);
+        return answer.save();
```

Comment:
It seems mongoose save() returns the inserted document. We could just use that as a response, so we can insert it on the front end.